docs/reference/troubleshooting/common-issues/high-cpu-usage.asciidoc

```diff
@@ -24,11 +42,13 @@ threads API>> to check for resource-intensive threads running on the node.
 
 [source,console]
 ----
-GET _nodes/my-node,my-other-node/hot_threads
+GET _nodes/hot_threads
 ----
 // TEST[s/\/my-node,my-other-node//]
 
-This API returns a breakdown of any hot threads in plain text.
+This API returns a breakdown of any hot threads in plain text. High CPU usage
+frequently correlates to <<task-queue-backlog,particular tasks and/or their
+backlog>>.
```

Comment:
[we can just use or](https://www.collinsdictionary.com/dictionary/english/inclusive-or) :)

```suggestion
This API returns a breakdown of any hot threads in plain text. High CPU usage
frequently correlates to <<task-queue-backlog,a long-running task, or a
backlog of tasks>>.
```